bugsnag-android-core/src/main/java/com/bugsnag/android/Event.java

```diff
@@ -575,4 +577,47 @@ void setRedactedKeys(Collection<Pattern> redactedKeys) {
     void setInternalMetrics(InternalMetrics metrics) {
         impl.setInternalMetrics(metrics);
     }
+
+    /**
+     * Returns the delivery strategy for this event, which determines how the event
+     * should be delivered to the Bugsnag API.
+     *
+     * @return the delivery strategy, or null if no specific strategy is set
+     */
+    @NonNull
+    public DeliveryStrategy getDeliveryStrategy() {
+        if (impl.getDeliveryStrategy() != null) {
+            return impl.getDeliveryStrategy();
+        } else {
+            if (getImpl().getOriginalUnhandled()) {
```

Comment:
nit: the `else` isn't required here since the `if (impl.getDeliveryStrategy() != null)` returns